install-dev/controllers/console/process.php

```diff
@@ -29,7 +29,6 @@
 class InstallControllerConsoleProcess extends InstallControllerConsole implements HttpConfigureInterface
 {
 
-    protected $model_database;
```

Comment:
You shouldn't delete this, it's used on line 40

Reply:
I did not delete it, moved it in the parent class. https://github.com/PrestaShop/PrestaShop/pull/8679/files#diff-509ec9c5ed185b146810b242732efaacR69

Reply:
Oh I missed that, my mistake 👍